### PR TITLE
refactor(ui): refactor ac signin exp

### PR DESCRIPTION
**Pull request by simeng-li**
## Summary

Refactor ac signin exp

1. Refactor the sign-exp API. Should not query user sign-in-exp data for AC app. Use hardcoded AC experience settings instead.
2. Add a slogan to AC sign-in-exp
3. Adjust the web sign-in page layout according to the latest design
image

Updates:
AC:
image
Rest:
image

## Testing

test locally
@logto-io/eng

### CODE REVIEW

File: packages/core/src/routes/well-known.test.ts [hunk 1]
```diff
@@ -112,13 +116,16 @@ describe('GET /.well-known/sign-in-exp', () => {
...
     expect(response.status).toEqual(200);
 
     expect(response.body).toMatchObject(
       expect.objectContaining({
-        ...mockSignInExperience,
-        signInMethods: adminConsoleSignInMethods,
+        ...adminConsoleSignInExperience,
+        branding: {
+          ...adminConsoleSignInExperience.branding,
+          slogan: 'admin_console.welcome.title',
```
Comment (gao-sun) [inline packages/core/src/routes/well-known.test.ts:127]: Welcome to Admin Console / 欢迎来到管理控制台

Comment (simeng-li) [reply packages/core/src/routes/well-known.test.ts:127]: test i18n key

Comment (gao-sun) [reply packages/core/src/routes/well-known.test.ts:127]: yes, see slack

File: packages/core/src/routes/well-known.ts [hunk 2]
```diff
@@ -46,30 +65,12 @@ export default function wellKnownRoutes<T extends AnonymousRouter>(router: T, pr
...
-        ctx.body = {
-          ...signInExperience,
-          socialConnectors,
-          notification: i18next.t('demo_app.notification'),
-        };
-
-        return next();
-      }
+      const notification =
+        interaction?.params.client_id === demoAppApplicationId
```
Comment (IceHe) [inline packages/core/src/routes/well-known.ts:69]: nit: maybe better extract `interaction?.params.client_id === demoAppApplicationId` to a local variable for reuse here and above.

```ts
      // Hard code AdminConsole sign-in methods settings.
      if (interaction?.params.client_id === adminConsoleApplicationId) {
```

File: packages/schemas/src/seeds/sign-in-experience.ts [hunk 3]
```diff
@@ -33,9 +33,11 @@ export const defaultSignInExperience: Readonly<CreateSignInExperience> = {
...
 
-export const adminConsoleSignInMethods = {
-  username: SignInMethodState.Primary,
-  email: SignInMethodState.Disabled,
-  sms: SignInMethodState.Disabled,
-  social: SignInMethodState.Disabled,
+export const adminConsoleSignInExperience: CreateSignInExperience = {
+  ...defaultSignInExperience,
+  branding: {
+    style: BrandingStyle.Logo_Slogan,
```
Comment (demonzoo) [inline packages/schemas/src/seeds/sign-in-experience.ts:39]: Why is this using `camel snake case`?

File: packages/ui/src/pages/SignIn/index.module.scss [hunk 4]
```diff
@@ -27,6 +27,14 @@
   }
 }
 
+.placeHolderTop {
+  flex: 3;
+}
+
+.placeHolderBottom {
```
Comment (demonzoo) [inline packages/ui/src/pages/SignIn/index.module.scss:34]: nit: placeholder is one word. Don't need to capitalize the letter `H`

File: packages/schemas/src/seeds/sign-in-experience.ts (hunk 3 above)
Comment (simeng-li) [reply packages/schemas/src/seeds/sign-in-experience.ts:39]: maybe some legacy issue, defined in schema like this.

File: packages/core/src/routes/well-known.test.ts (hunk 1 above)
Comment (simeng-li) [reply packages/core/src/routes/well-known.test.ts:127]: content updated

File: packages/ui/src/pages/SignIn/index.module.scss (hunk 4 above)
Comment (simeng-li) [reply packages/ui/src/pages/SignIn/index.module.scss]: updated

**Review by gao-sun — APPROVED** [review]

**Pull request merged**